src/fast-path.js

```diff
@@ -364,6 +372,12 @@ FPp.needsParens = function(assumeExpressionContext) {
           return false;
 
         case "ExpressionStatement":
+          if (
```

Comment:
Tests pass without this addition. It shouldn't be necessary given the `if` at the top, either, if I'm reading them both correctly. Should either get rid of this or, if it's actually necessary, add a test which depends on it.